tests/utils/util.py

```diff
@@ -79,6 +82,14 @@ def get_connection(adapter, name="_test"):
         conn = adapter.connections.get_thread_connection()
         yield conn
 
+# Uses:
```

Comment:
Why do we need this documentation?

Reply:
This is just copied from the `util.py` dbt ships. Should I remove it?

Reply:
Let's remove it. It seems pretty useless.

Reply:
Done